### DISCUSSION TITLE
How to customize/add media queries?

### DISCUSSION THREAD

**Discussion by haseeb5i** [heroui-inc/heroui#477] [Help]
There is an option to override/add new breakpoints but these are not reflected in already available media queries. I want to use different breakpoints in my app, so i was wondering how can i achieve that instead of manually writing media query everytime i need to use them.

For example when using responsive `Grid` component, all media queries are applied on default breakpoints provided by nextui but i want this component to use my custom defined breakpoints.

Any info or pointer would be a great help. Thanks.

**Comment by jrgarciadev** [comment]
Hey guys, we are working on a solution to make the stitches media breakpoints externally configurable, we hope to include it in the next version

**Reply by dialformonkey** [reply]
Any update on the extend capability of the media breakpoints?

**Reply by aficiomaquinas** [reply]
I'm very interested in this. It's quite difficult to integrate NextUI to my scenario without the ability to customize media queries.

**Comment by jrgarciadev** [comment]
Hey guys sorry for the delay, please update to [V2](https://nextui.org/blog/nextui-v2) and use TailwindCSS config to override breakpoints https://tailwindcss.com/docs/responsive-design#using-custom-breakpoints
